### PR TITLE
add plugin release date

### PR DESCRIPTION
**Pull request by cmeriaux**
Add plugin release date: 
- json key: release-date
- type: string
- pattern: \\d\\d\\d\\d-\\d\\d-\\d\\d

Type date seems not to be supported yet until json schema draft 7.

### CODE REVIEW

**Comment by cmeriaux** [comment]
fixes #33

**Comment by cmeriaux** [comment]
Another PR will be submited in Notepad++

image

File: validator.py [hunk 1]
```diff
@@ -50,13 +53,17 @@ def parse(filename):
...
     for error in schema.iter_errors(pl):
+        pprint(error.absolute_path)
         post_error(error.message)
 
     foldernames = []
     displaynames = []
     repositories = []
 
-    os.mkdir("./" + bitness_from_input)
+    path = Path(bitness_from_input)
```
Comment (chcg) [inline validator.py]: Why is this change needed?

File: src/pl.x64.json [hunk 2]
```diff
@@ -11,7 +11,8 @@
 			"repository": "https://github.com/jcaillon/3P/releases/download/v1.8.4/3P_x64.zip",
 			"description": "[Requires .net framework 4.6.1]\r\n3P is a notepad++ plug-in designed to help writing OpenEdge ABL (formerly known as Progress 4GL) code. It provides :\r\n- a powerful auto-completion\r\n- tool-tips on every words\r\n- a code explorer to quickly navigate through your code\r\n- a file explorer to easily access all your sources\r\n- the ability to run/compile and even PROLINT your source file with an in-line visualization of errors\r\n- more than 50 options to better suit your needs\r\n- and so much more!\r\nVisit http://jcaillon.github.io/3P/ for more details on the plugin",
 			"author": "Julien Caillon",
-			"homepage": "https://jcaillon.github.io/3P/"
+			"homepage": "https://jcaillon.github.io/3P/",
+			"release-date": "2019-02-23"
```
Comment (chcg) [inline src/pl.x64.json]: Have you tested also the existing implementation of N++7.7 how it behaves with this additional field value? Is it just ignored?

File: validator.py (hunk 1 above)
Comment (cmeriaux) [reply validator.py]: I've just add a piece of code to clean up existing folder. It's useful when running several times the validator.
If you prefer, I can remove every changes in validator.py

File: src/pl.x64.json (hunk 2 above)
Comment (cmeriaux) [reply src/pl.x64.json]: Yes. I've test with N++ 7.7, the new field is just ignored.
